index.html

```diff
@@ -0,0 +1,179 @@
+<!DOCTYPE html>
+<html lang="en">
+<link rel="stylesheet" href="C:\Users\Genaro\Desktop\css\bootstrap.css">
+<link rel="stylesheet" href="C:\Users\Genaro\Desktop\css\bootstrap-grid.css">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>GirlDevelopIt</title>
+</head>
+
+<body>
+    <br>
+    <div id="cuerpo" class="container-fluid">
+
+        <div id="top" class="container-fluid">
+            <header>
+                <img src="https://stackoverflow.blog/wp-content/uploads/2017/08/wordmark-tagline-pink-2.png" class="img-fluid" alt="Responsive image"
+                    width="400px">
+            </header>
+        </div>
+        <br>
+        
+        <div id="About" class="container-fluid">
+            <h1 id="bruh1">About</h1>
+        </div>
+
+        <div class="row">
+            <div class="col-md-6">
+                <p class="myclass">
+                    It can be indimidating for women to learn and ask questions when they are in an
+                    <b>extreme minority</b>. While open and welcoming ,today's budding developer community is up to
+                    <b>91% male.</b> If we empower more females with the confidence in their technological capabilities
+                    <i>we can begin to change this landscape.</i>
+                </p>
+                <br>
+            </div>
+
+        </div>
+        <div>
+            <h1>Our Locations</h1>
+        </div>
+
+        <div>
+            <ul>
+                <li>USA</li>
+                <ul>
+                    <li>
+                        <a href="https://www.girldevelopit.com/chapters/new-york">New York</a>
+                    </li>
+                    <li>
+                        <a href="https://www.girldevelopit.com/chapters/columbus">Columbus</a>
+                    </li>
+                    <li>
+                        <a href="https://www.girldevelopit.com/chapters/austin">Austin</a>
+                    </li>
+                    <li>
+                        <a href="https://www.girldevelopit.com/chapters/philadelphia">Philly</a>
+                    </li>
+                </ul>
+            </ul>
+            <ul>
+                <li>World</li>
+                <ul>
+                    <li>
+                        <a href="ottawa.html">Ottawa</a>
+                    </li>
+                    <li>
+                        <a href="sydney.html">Sydney</a>
+                    </li>
+                </ul>
+            </ul>
+
+        </div>
+
+        <div>
+            <h1>Upcoming Classes</h1>
+        </div>
+
+        <div class="row">
+            <div class="col-md-8">
+                <table border="1" style="width: 80%">

```

Comment:
Don't use border directly as an attribute, use css `border` instead https://www.w3schools.com/css/css_border.asp
Don't use inline styling, always reference css in classes for external files.
Using `width: 80%` is unnecessary, since it won't work will all devices. To have different width sizes, use bootstrap columns instead https://getbootstrap.com/docs/4.0/layout/grid/